README.md

```
@@ -0,0 +1,191 @@
+
+---
+
+## 👥 팀원
+
+<table>
+    <tr align="center">
+        <td colspan="2"><b>Frontend</b></td>
+    </tr>
+    <tr align="center">
+        <td>
+            <img src="https://avatars.githubusercontent.com/u/86469788?v=4?size=100" width="100">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)